Test2

```diff
@@ -0,0 +1,148 @@
+    # all text fields are joined.
+    description_in = Identity()
+    description_out = NormalizedJoin()
+    #requirements_in = Identity()
+    #requirements_out = NormalizedJoin()
+    #skills_in = Identity()
+    #skills_out = NormalizedJoin()
+    #benefits_in = Identity()
+    #benefits_out = NormalizedJoin()
+
+
+REF_REGEX = re.compile(r'\/(\d+)$')
+
+APPEND_GB = lambda x: x.strip() + ", GB"
+
+
+class SimplyLawJobs(CrawlSpider):
+    """ Should navigate to the start_url, paginate through
+    the search results pages and visit each job listed.
+    For every job details page found, should produce a JobItem
+    with the relevant fields populated.
+
+    You can use the Rule system for CrawSpider (the base class)
+    or you can manually paginate in the "parse" method that is called
+    after the first page of search results is loaded from the start_url.
+
+    There are some utilities above like "NormalizedJoin" and JobItemLoader
+    to help making generating clean item data easier.
+    """
+    
+    start_urls = ["http://www.simplylawjobs.com/jobs"]
+    name = 'lawjobsspider'
+    
+    
+    #----------------#
+    #----------------#
+    #-YOUR CODE HERE-#
+    #----------------#
+    #----------------#
+
+    
+    
+    rules = (Rule (LinkExtractor(allow=(),restrict_xpaths=('//*[@id="search_results_list"]/ul/li[1]',))
+    , callback="parse_items", follow= True,),)
+
+    def parse_Items(self, response):
+        hxs = HtmlXPathSelector(response)
+        titles = hxs.select("//html")
+        Items = []
+        
+        for titles in titles:
+            Item = JobItem()
+      # required fields
+            Item ["title"]   = titles.select("//*[@id="search_results_list"]/ul/li[1]/div[2]/div[1]/a[1]/h2").extract()
+            Item ["company"] = titles.select("//*[@id="search_results_list"]/ul/li[1]/div[2]/div[1]/a[2]").extract()
+            Item ["JobUrl"]  = titles.select("//*[@id="search_results_list"]/ul/li[1]/div[2]/div[2]/a[1]").extract()
+      #Could Not Find the Job ID on the Webpage today {Item ["JobId"]}
+      
+      #Optional fields
+            Item ["location"]   = titles.select("//*[@id="search_results_list"]/ul/li[1]/div[2]/div[1]/p/span").extract()
+            Item ["salary"]     = titles.select("//*[@id="search_results_list"]/ul/li[1]/div[2]/div[1]/p/text()").extract()
+            Item ["desciption"] = titles.select("//*[@id="search_results_list"]/ul/li[1]/p[1]/text()").extract()
+            Item ["JobType"]    = titles.select("//*[@id="center_column"]/div[2]/div[2]/text()[4]").extract()
+            Item ["JobSector"]  = titles.select("//*[@id="center_column"]/div[2]/div[2]/text()[5]").extract()
+
+            Items.append(Item)
+            return(Items)
+
+
+    #Using Item Loaders
+    def JobItemLoader(self, response1):
+        loader = MyItemLoader(selector=response1)
+        #loader.add_xpath('title', "//*[@id="search_results_list"]/ul/li[1]/div[2]/div[1]/a[1]/h2")
```

Comment:
can you uncomment this and make it work? You also don't need to define a new JobItemLoader method. You just need to create an instance of it and then use the api.